mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -400,65 +426,68 @@ def _compute_moc_postprocess(config, streams, variableMap, calendar,
-        if not os.path.exists(outputFileTseries):
-            for nm in range(12):
-                print '     month = ', nm + 1
-                horizontalVel = ds.avgNormalVelocity[ncount, :, :].values
-                verticalVel = ds.avgVertVelocityTop[ncount, :, :].values
-                velArea = verticalVel * areaCell[:, np.newaxis]
-                transportZ = _compute_transport(maxEdgesInTransect,
-                                                transectEdgeGlobalIDs,
-                                                transectEdgeMaskSigns,
-                                                nVertLevels, dvEdge,
-                                                refLayerThickness,
-                                                horizontalVel)
-                mocTop = _compute_moc(latAtlantic, nVertLevels, latCell,
-                                      regionCellMask, transportZ, velArea)
-                mocAtlantic26[ncount] = np.amax(mocTop[:, indlat26])
-                ncount += 1
-
-            Time[ncount-12:ncount] = ds.Time[ncount-12:ncount].values
-
-            # Save to file
-            print '   save to yearly file...'
-            ncFile = netCDF4.Dataset(outputFileTseries, mode='w')
-            # create dimension
-            ncFile.createDimension('Time', 0)
-            # create variables
-            Timevar = ncFile.createVariable('Time', 'f4', ('Time',))
-            Timevar.description = 'days since start of simulation'
-            mocAtlantic26var = ncFile.createVariable('mocRapidArrayMax',
-                                                     'f4', ('Time',))
-            mocAtlantic26var.description = 'Max MOC Atlantic streamfunction'\
-                                           ' nearest to RAPID Array latitude'\
-                                           ' (26.5N)'
-            mocAtlantic26var.units = 'Sv (10^6 m^3/s)'
-            # save variables
-            Timevar[:] = Time[ncount-12:ncount]
-            mocAtlantic26var[:] = mocAtlantic26[ncount-12:ncount]
-            ncFile.close()
-        else:
-            ncount += 12
-            # Read from file
-            print '     Read previously computed Atlantic MOC at 26.5N'\
-                  ' from file...'
-            ncFile = netCDF4.Dataset(outputFileTseries, mode='r')
-            Time[ncount-12:ncount] = ncFile.variables['Time'][:]
-            mocAtlantic26[ncount-12:ncount] = ncFile.variables['mocRapidArrayMax'][:]
-            ncFile.close()
-
-    # Store max Atlantic MOC at 26.5 to dictionary
-    mocDictTseries = {'Time': {'dims': ('Time'), 'data': Time},
-                      'mocRapidArrayMax': {'dims': ('Time'),
-                                           'data': mocAtlantic26}}
-
-    return (mocDictClimo, mocDictTseries)
+    outputFileTseries = '{}/mocTimeSeries.nc'.format(outputDirectory)
+
+    continueOutput = os.path.exists(outputFileTseries)
+    if continueOutput:
+        # read in what we have so far
```

Comment:
I would add a print statement here:
print '   Read in previously computed MOC time series'